torch/_dynamo/convert_frame.py

```diff
@@ -1035,16 +1036,6 @@ def format_guard_failures() -> str:
                 graph_op_count = output.count_calls()
                 graph_node_count = len(output.graph.nodes)
                 graph_input_count = len(output.placeholders)
-                entire_frame_compile_time = frame_phase_timing[frame_key].get(
```

Comment:
Note that I'm removing obsolete columns in this PR. For example, I'm just capturing `dynamo_cumulative_compile_time_us` and removing `entire_frame_compile`. Same for the other three metrics affected here.

Reply:
IMO don't remove the obsolete columns yet. In fact, keep populating them via legacy code. There may be tests relying on them and you want to hose that down separately. The important thing is Scuba entries no longer read from these timings.